src/hooks/useAllTickets.ts

```diff
@@ -1,19 +1,16 @@
 import { ServiceRequestSummary } from '@types';
-import { useSearchParams } from 'next/navigation';
 import { useEffect, useState } from 'react';
 import * as DataService from '@services/DataService';
 
-export default function useRecentTickets() {
+export default function useAllTickets() {
   const [tickets, setTickets] = useState<(ServiceRequestSummary)[]>([]);
-  // Hack:  using url change to trigger to get recent tickets
-  const params = useSearchParams();
```

Comment:
What is the impact of getting rid of this code? Does it mean that the recent tickets table is no longer being updated when a new ticket is added?

Reply:
Good catch, this hack to trigger a refresh needs to be replaced with a SWR mutate call inside of the dialog that creates the tickets. Will update that in a new commit.

Reply:
Whoops nevermind, its the other PR that has SWR, so I'll revert this change and fix in the other PR